sources/packages/backend/libs/test-utils/src/factories/offering-change-request.ts

```diff
@@ -0,0 +1,31 @@
+import { EducationProgramOffering, OfferingStatus } from "@sims/sims-db";
+
+/**
+ * Create a fake offering request change.
+ * @param relations dependencies:
+ *  - `currentOffering` current offering, which is requested for the changed.
+ * @returns the current and requested offering.
+ */
+export function createFakeOfferingRequestChange(relations: {
+  currentOffering: EducationProgramOffering;
+}): EducationProgramOffering[] {
```

Comment:
The return type can be an object because only upto 3 offerings can be a return for this method and will be easy for the consumer to precisely read the offerings.

```ts
Promise<{requestedOffering:EducationProgramOffering, precedingOffering:EducationProgramOffering,parentOffering?:EducationProgramOffering}>
```